reliability-v2/README.md

```diff
@@ -221,13 +221,15 @@ The following funcs are supported now:
 | new_project | number of projects | developer | Create n projects for the user|
 | check_all_projects  | N/A | developer | Check projects under the user|
 | new_app  | number of projects | developer | New an app under each project|
-| load_app  | number of projects number of clients | developer | Load an app under each project with a number of clients|
+| load_app  | number of projects  number of clients | developer | Load an app under each project with a number of clients|
+| apply  | number of projects  file location | all | apply a file|
```

Comment:
looks like there are 2 "apply"s in the list. do both work? Are they used under different sections that we need to section off this list?

Reply:
Aha, thanks! my last change was not saved on local. I have fixed this, with some other little changes to README.md.